test/lib/test-server.js

```diff
@@ -0,0 +1,56 @@
+var request = require('request')
+var createApp = require('../../index.js')
+
+var portCounter = 10000
+
+module.exports = function (cb) {
+  // setup config
+  // =
+
+  var config = {
+    township: {
+      secret: 'very very not secret',
+      memdb: true,
```

Comment:
Oh that's a good point. Yeah it's just for tests. I'll switch it over to passing in objects